src/tokenizer.rs

```diff
@@ -958,6 +1014,65 @@ mod tests {
         compare(expected, tokens);
     }
 
+    #[test]
+    fn tokenize_postgresql_bitwise_operations() {
```

Comment:
I think it's usually enough to test the parser. Is there any point to the tokenizer-specific tests?